lib/screens/categorylisting.dart

```diff
@@ -627,7 +629,7 @@ class _CategoryListingState extends State<CategoryListing> {
     });
     var response;
     print(sortBy);
-    if (sortBy != null) {
+    if (sortBy != null && sortBy.length > 0) {
```

Comment:
here `if(sortBy)` this would be sufficient.

Reply:
if(sortBy) is not sufficient. sortBy is not bool. I just wanted to make sure for a null check on sortBy.